src/Microsoft.AspNetCore.NodeServices/Microsoft.AspNetCore.NodeServices.csproj

```diff
@@ -1,29 +1,14 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
-  <Import Project="..\..\build\common.props" />
-
   <PropertyGroup>
     <Description>Invoke Node.js modules at runtime in ASP.NET Core applications.</Description>
     <TargetFramework>netstandard2.0</TargetFramework>
-    <PackageTags>aspnetcore;aspnetcoremvc;nodeservices</PackageTags>
-    <TypeScriptCompileBlocked>true</TypeScriptCompileBlocked>
-    <GenerateDocumentationFile>true</GenerateDocumentationFile>
   </PropertyGroup>
 
   <ItemGroup>
-    <None Remove="node_modules\**\*" />
```

Comment:
I'm not keen on this or any of the following lines in this `csproj` being moved out into `Directory.Build.props`, because:

1. It's hard to reason about which `node_modules` or `Content` files are being included/excluded when the paths aren't even relative to the file that's referencing them
2. It makes an assumption about conventions for inclusion/exclusion and prepublishing always being the same across every project in `src`, which isn't guaranteed. Although they might all happen to follow the same conventions today, there's no reason to believe that should always be so. Not all of them should necessarily have a `Content` directory, use `node_modules`, etc.

So I would prefer not to deduplicate these values, but rather keep the `src` projects equivalently decoupled as they were before.

Reply:
Sounds good. I'll revert this piece of the change.